frontend/src/pages/TaskOverviewPage.tsx

```diff
@@ -87,7 +122,17 @@ export const getTaskOverviewData = (task: Task, editable: boolean) => {
         editable: false,
       },
     ],
+    [
+      {
+        label: i18n.t('Last modified by'),
+        keyName: 'lastUpdatedByUserId',
+        value: editedAtText,
+        format: 'updatedAt',
```

Comment:
Does this have corresponding styles defined?
If not, this could just be left out